certified-connectors/LatinShare SHP Management/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1060 @@
+{
```

Comment:
@NicolasValenzuela This swagger file is not up to date, can you please update it with the latest version.